com.writerduet.writersolo.yaml

```diff
@@ -0,0 +1,49 @@
+app-id: com.writerduet.writersolo
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+command: writersolo
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=x11
```

Comment:
You don't need both. If this is electron, add x11 only and make wayland opt out.